src/test/java/lotto/domain/WinningLottoTest.java

```diff
@@ -8,32 +8,32 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
-public class WinningNumberTest {
+public class WinningLottoTest {
 
     @Test
     @DisplayName("생성자 검증 - 당첨 번호 null")
-    void constructorDIExceptionForWinningNumberNull() {
-        assertThatThrownBy(() -> new WinningNumber(null, LottoNumber.lottoNumber(7))).isInstanceOf(IllegalArgumentException.class);
+    void constructorDIExceptionForWinningLottoNull() {
+        assertThatThrownBy(() -> new WinningLotto(null, LottoNumber.lottoNumber(7))).isInstanceOf(IllegalArgumentException.class);
     }
```

Comment:
```suggestion
    void constructorDIExceptionForWinningLottoNull() {
        assertThatThrownBy(() -> new WinningLotto(null, LottoNumber.lottoNumber(7)))
                .isInstanceOf(IllegalArgumentException.class);
    }
```

1. 적절한 줄바꿈으로 가독성 향상
2. 테스트 메서드 이름에서 사용된 DI라는 부적절한 표현
3. `@DisplayName` 설명의 모호함

포인트가 여럿 있군요 😉